samples: zigbee: Fix nRF53 DFU by applying correct sysbuild config (#17796)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`zigbee_light_switch_sample` sample:

  * Added the option to configure transmission power.

>   * Fixed FOTA operation on nRF5340 by correcting configuration.

Reviewer: ```suggestion
  * Fixed the FOTA configuration for the nRF5340 DK.
```

Author: 👍 fixed
